pkg/cache/context.go

```diff
@@ -365,19 +366,68 @@ func (ctx *Context) bindPodVolumes(pod *v1.Pod) error {
 				zap.String("podName", pod.Name))
 		} else {
 			log.Logger().Info("Binding Pod Volumes", zap.String("podName", pod.Name))
-			boundClaims, claimsToBind, _, err := ctx.apiProvider.GetAPIs().VolumeBinder.GetPodVolumes(assumedPod)
+			boundClaims, claimsToBind, unboundClaimsImmediate, err := ctx.apiProvider.GetAPIs().VolumeBinder.GetPodVolumes(assumedPod)
```

Comment:
This same call is made in `AssumePod` line 457 why do we not need the change there?

Reply:
We probably do. I missed that one.

Reply:
actually, i don't think we do. This logic will have already run via the PreFilter() / Filter() calls on the upstream VolumeBinder plugin (we call these as part of node fitness checks).